opencti-platform/opencti-graphql/src/database/redis.ts

```diff
@@ -171,9 +171,15 @@ const delKeyWithList = async (keyId: string, listIds: string[]) => {
   await Promise.all([keyPromise, ...listsPromise]);
 };
 const setKeyWithList = async (keyId: string, listIds: string[], keyData: any, expirationTime: number) => {
-  const keyPromise = getClientBase().set(keyId, JSON.stringify(keyData), 'EX', expirationTime);
-  const listsPromise = listIds.map((listId) => setInList(listId, keyId, expirationTime));
-  await Promise.all([keyPromise, ...listsPromise]);
+  // Wrapped in try/catch fixed an issue with invalid expiration time in set
+  try {
```

Comment:
Do you have repro case for that part that is not related to the feature?

Reply:
No, I was sporadically getting redis issues and this was the only thing that staved it off for good. I wasn't sure what exactly what was causing it. Regardless, I believe it's best to wrap code that can throw errors in a try/catch block.
I last got this error maybe 5 months ago so I can't show you what exactly it was, but I can guarantee I haven't had it appear since.